webapp/graphql/Moderation.js

```diff
@@ -82,9 +82,6 @@ export const reportsListQuery = () => {
             name
             disabled
             deleted
-            followedByCount
-            contributionsCount
-            commentedCount
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-09T17:07:42Z" title="Monday, December 9th 2019, 6:07:42 pm +01:00">Dec 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -82,9 +82,6 @@ export const reportsListQuery = () => {
             name
             disabled
             deleted
-            followedByCount
-            contributionsCount
-            commentedCount
```


Awww yeah. That's what I wanted to do in my next PR. I'm going to rebase my code on that.